code/game/objects/items/weapons/switchtool.dm

```diff
@@ -59,6 +62,9 @@
 		choose_deploy(user)
 
 /obj/item/weapon/switchtool/attackby(var/obj/item/used_item, mob/user)
+	if(istype(used_item, /obj/item/weapon/screwdriver) && no_remove)
```

Comment:
I had people saying, to me, what if I remove the sword and shield to me in discord. So it clearly wasn't clear enough you cant remove light.